gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/tasks/CreateManifestTaskTest.groovy

```diff
@@ -1,10 +1,156 @@
 package com.palantir.gradle.dist.tasks
 
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.palantir.gradle.dist.GradleTestSpec
+import nebula.test.dependencies.DependencyGraph
+import nebula.test.dependencies.GradleDependencyGenerator
 import org.gradle.api.Project
 import org.gradle.testfixtures.ProjectBuilder
-import spock.lang.Specification
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.TaskOutcome
 
-class CreateManifestTaskTest extends Specification {
+import java.nio.file.Files
+import java.nio.file.StandardCopyOption
+
+class CreateManifestTaskTest extends GradleTestSpec {
+
+    def jsonMapper = new ObjectMapper()
```

Comment:
can we ask the `CreateManifestTask` class to create an objectmapper to not duplicate the code?